ivy/functional/frontends/paddle/tensor/random.py

```diff
@@ -7,6 +7,15 @@
 )
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def exponential_(x, lam=0.1, name=None):
```

Comment:
```suggestion
def exponential_(x, lam=1.0, name=None):
```
default appears to be 1.0 https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/Tensor/exponential__en.html#exponential

Reply:
@lucasalavapena  Actually I couldnt able to do that, I have no idea how the commits on the other branch got linked with this and I also couldnt able to remove this. as this commit isnt available in this branch's git log. Any suggestions?

Reply:
@dhanush-2501 I see you made a new PR, I have assigned myself to it. You could have reset your head and done a force push to your fork.